src/main/java/gregtech/common/metatileentities/storage/CachedRecipeData.java

```diff
@@ -129,11 +88,22 @@ public boolean getIngredientEquivalent(int slot) {
 
     private boolean simulateExtractItem(ItemStackKey itemStack) {
         int amountToExtract = requiredItems.getOrDefault(itemStack, 0) + 1;
```

Comment:
Shouldn't this just default to 1 instead of defaulting to zero and then adding one? Always adding one would throw off the amount stored in `requiredItems` as well.